tox.ini

```
@@ -168,3 +170,16 @@ passenv =
 skip_install = false
 allowlist_externals =
   git
+
+[testenv:changelog-draft]
+deps =
+    towncrier
+    -r{toxinidir}/docs/requirements.txt
```

I just noticed that this has inconsistent indents. I standardized on two-space indents for ini-files.
```suggestion
  towncrier
  -c{toxinidir}/docs/requirements.txt
```

---

I can make sure to use 2-ist spacing. The file is already a little inconsistent, I think, so it wasn't clear which way to go. We'll do 2-ist for the new stuff and I'll standardize it in a PR sometime later.